src/index.js

```diff
@@ -93,6 +93,7 @@ class App extends React.Component {
   componentDidMount = () => {
     const { actions } = this.props;
     actions.getApiKeys();
+    actions.getAdmin();
```

Comment:
getAdmin should only be called when the user is logged in, put a check here @AakashMallik

Reply:
@fragm3 Will do. Btw, I noticed that in your code in Susi.Chat, you passed access_token explicitly in the getAdminApi. Was that needed coz ajax.js would put the "access_token" value anyway. Let me know as I have a pending PR, so I can make the changes before finalizing it.

Reply:
@AakashMallik we will need to pass the access token separately on login, as the access token is not set till then(async in nature)